app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -315,4 +317,8 @@ public static String formLoadPayloadStatus() {
     public static boolean isDetailTabSwipeActionEnabled() {
         return doesPropertyMatch(DETAIL_TAB_SWIPE_ACTION_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
     }
+
+    public static boolean isOfflineUpdateEnabled() {
+        return doesPropertyMatch(OFFLINE_UPDATE_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
```

Comment:
I currently have the default value set as Enabled, because I find it to be a pain to enable developer options, and because this is well-hidden as it is (in the options menu of the `UpdateActivity`), but would be open to switching it if that sounds wrong